projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -351,11 +329,9 @@ public function add_appearance_menu( $wp_admin_themes = false, $wp_admin_customi
 
 	/**
 	 * Adds Plugins menu.
-	 *
-	 * @param bool $wp_admin Optional. Whether links should point to Calypso or wp-admin. Default false (Calypso).
 	 */
-	public function add_plugins_menu( $wp_admin = false ) {
-		if ( $wp_admin ) {
+	public function add_plugins_menu() {
```

Comment:
This a tricky one too. Since `plugin-install.php` and `plugin-editor.php` do not have a Calypso page to switch back do you think it would be better hiding / showing them depending on Simple / Atomic?

Reply:
We already do that. Simple sites overrides the preferred view to `default` and Atomic sites overrides the preferred view to `classic`.